mods/cnc/rules/defaults.yaml

```diff
@@ -480,10 +480,13 @@
 		SellSounds: cashturn.aud
 	Capturable:
 	WithMakeAnimation:
+		MakeUpgrades: underConstruction
+	DisableOnUpgrade:
+		UpgradeTypes: underConstruction
```

Comment:
In that case
>we should *fix* what already exists if it's not good enough to reuse 

Anyway, at this point does it really hurt to add *yet another one*?